### DISCUSSION TITLE
Can the docker version of the server and client be officially launched?

### DISCUSSION THREAD

**Discussion by taipinghehe** [SoftEtherVPN/SoftEtherVPN#1543] [Q&A]
The software softether is very good, and I would like to express my deep gratitude to the engineers of the software. 
I am currently using a QNAP NAS server. I'd like to be able to use the docker version of the softether server and client on top of it. In this way, if there is a problem or there is a change, you can delete the docker container directly without affecting the system. Hope to release the docker version soon. thanks.

**Comment by chipitsine** [comment]
there's   [howto](https://github.com/SoftEtherVPN/SoftEtherVPN/wiki/Container-deployment-HowTo)  from @weidi

**Reply by weidi** [reply]
Nevertheless i would suggest launching an official image

**Reply by chipitsine** [reply]
I was about adding mentioned link to README as "official", btw

(still somewhere in New Year holidays)

is there something wrong with that doc ? some better doc ?

**Reply by weidi** [reply]
I think the request would be to have an image that is officially maintained (and hosted in SoftetherVPN "owned" repository on GitHub and dockerhub). Would make it look more professional.

**Reply by chipitsine** [reply]
It is something that may happen in the future. Currently, mentioned doc hopefully is the best possible

**Reply by taipinghehe** [reply]
Thanks a lot for the reply. I read the howto link, the docker server version update is not bad, but the latest one I can find on the dockerhub client version is a year ago, and everyone doesn't pay much attention to the docker client version.
My company's qnap nas does not have a public network ip address, but my home has a public network ip. I plan to install the softether client version of docker on the nas to connect to the softether server at my home. So I hope the softether client version of docker can be updated in time, it would be great if it can be maintained by the official.

**Reply by weidi** [reply]
What client image are you refering to that is outdated? I just pushed toprock/softether:vpnclient if this is going to help in any way.

**Reply by taipinghehe** [reply]
Found this docker image, thank you very much. If I encounter any difficulties, I will ask you again.
